include/aikido/control/ros/RosPositionCommandExecutor.hpp

```diff
@@ -0,0 +1,75 @@
+// Assume it has 4 positions
+#ifndef AIKIDO_CONTROL_ROS_ROSPOSITIONCOMMANDEXECUTOR_HPP_
+#define AIKIDO_CONTROL_ROS_ROSPOSITIONCOMMANDEXECUTOR_HPP_
+#include <chrono>
+#include <future>
+#include <mutex>
+#include <ros/ros.h>
+#include <ros/callback_queue.h>
+#include <sensor_msgs/JointState.h>
+#include <pr_control_msgs/SetPositionAction.h>
+#include <actionlib/client/action_client.h>
+#include <aikido/control/PositionCommandExecutor.hpp>
+#include <Eigen/Dense>
+
+namespace aikido{
+namespace control {
+namespace ros {
+
+/// This class sends commands for the hand position to the ROS server
+class RosPositionCommandExecutor : public aikido::control::PositionCommandExecutor
+{
+public:
+  /// Constructor
+  /// \param[in] node ROS node handle for action client.
+  /// \param[in] serverName Name of the server to send traejctory to.
+  /// \param[in] connectionTimeout Timeout for server connection.
+  /// \param[in] connectionPollingPeriod Polling period for server connection.
+  RosPositionCommandExecutor(
+    ::ros::NodeHandle node,
+    const std::string& serverName,
+    const std::vector<std::string>& jointNames,
+    const std::chrono::milliseconds connectionTimeout = std::chrono::milliseconds{1000},
+    const std::chrono::milliseconds connectionPollingPeriod = std::chrono::milliseconds{20}
```

Comment:
Nit: Const is meaningless on a parameter passed by value in a function declaration.